### PR TITLE
Renjani - Bio 

### PR DESCRIPTION
**Pull request by renjani2022**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

File: student-bios/renjani2022.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Introduction
+
+<p>&nbsp;</p>
```
Comment (bermarte) [inline student-bios/renjani2022.md:3]: don't use HTML tag, or `<p>&nbsp;</p>` when writing HTML you can open a `<p>` tag and write some text inside of it. `<p>` is when you want to write a paragraph of text.

**Review by bermarte — CHANGES_REQUESTED** [review]
Once you change it, you will be able to merge your PR.

File: student-bios/bibhasingh.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Hello :wave:
```
[the remark this answers is not in the document]
Comment (renjani2022) [reply student-bios/bibhasingh.md:1]: thanks a lot for the notification @bermarte. I don't know why this file appears and attached to my file, I tried to delete from VSC, hope it works. The problem after I edited my file (deleting the html code), I can't push it again. My git code was rejected many times. Do you have any suggestion?

**Comment by bermarte** [comment]
Run the scripts locally and try to fix the errors, then add your .md file and push it again. To get rid of the file, you should remove it by running:
`git rm <file>`
then add it and push the change in your branch.
this time you should run
`git add .`
then commit and push it.

**Pull request closed without merging**